ui/src/app/shared/pipes/safeHtml.pipe.ts

```diff
@@ -1,10 +1,23 @@
 import {Pipe, PipeTransform} from '@angular/core';
 import {DomSanitizer} from '@angular/platform-browser';
+declare var sanitizeHtml: any;
 
 @Pipe({ name: 'safeHtml'})
 export class SafeHtmlPipe implements PipeTransform  {
   constructor(private sanitized: DomSanitizer) {}
-  transform(value) {
-    return this.sanitized.bypassSecurityTrustHtml(value);
+  transform(value: string, sanitize: boolean) {
+    if (!sanitize) {
```

Comment:
Maybe sanitize should not be optional for safeHtml

Reply:
The main goal here is to not do the process twice. In the ansi_up lib when we active the escape html it already do that. It's just for perf. But I'll change the default behavior in order to sanitize by default